cc1101.cpp

```diff
@@ -18,8 +18,8 @@ uint8_t cc1101::revision = 0x01;
 0x00, // 0A CHANNR   
```

Comment:
In Zeile 12 ist wohl ein Zeilenumbruch abhanden gekommen:
0x47, // 03 FIFOTHR          RX filter bandwidth = 325 kHz, FIFOTHR = 0x470xD3, // 04 SYNC1
sollte besser wieder so aussehen:
0x47, // 03 FIFOTHR          RX filter bandwidth = 325 kHz, FIFOTHR = 0x47
0xD3, // 04 SYNC1

Reply:
Daran hatte ich ziemlich zu kämpfen, bis ich gesehen hatte, dass dadurch Einträge fehlen. :(

Ich hab es jetzt mal etwas aufgehübscht. Code